Scripts/whitesource-spdx.yml

```diff
@@ -1,4 +1,4 @@
-name: WhiteSource scan and SPDX report 
```

Comment:
We may need to rename this filename as well

Reply:
lets leave the name as new